Dockerfile

```diff
@@ -0,0 +1,28 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+FROM golang:alpine AS base
+COPY . /go/src/github.com/open-telemetry/opentelemetry-go/
```

Comment:
Just a nit: it really doesn't need to be in a properly formed GOPATHesque path. `COPY . /otel-go/` should work fine as well. The binaries installed with `go install` should still be somewhere in `/go/bin`.

Reply:
Got it. Just following [suggested best practice](https://blog.golang.org/docker).

Reply:
Yeah, that was a best practice 6 years ago. ;)

Reply:
Or just to be more precise - 6 years ago you were expected to put the project you were working on in GOPATH, otherwise stuff was sometimes breaking, not building or else. Now GOPATH is mostly for storing the versioned deps.

Anyway, it's not important. I see that the moved Dockerfiles ended up using this long GOPATH too, and it's not a fight worth having.